tests/system/src/test/java/test/javafx/scene/control/TabPaneHeaderLeakTest.java

```diff
@@ -92,16 +93,8 @@ public void testTabPaneHeaderLeak() throws Exception {
             tabPane.getTabs().clear();
             root.getChildren().clear();
         });
-        for (int i = 0; i < 10; i++) {
-            System.gc();
-            if (tabWeakRef.get() == null &&
-                textFieldWeakRef.get() == null) {
-                break;
-            }
-            Util.sleep(500);
-        }
-        // Ensure that Tab and TextField are GCed.
-        Assert.assertNull("Couldn't collect Tab", tabWeakRef.get());
-        Assert.assertNull("Couldn't collect TextField", textFieldWeakRef.get());
+
+        JMemoryBuddy.assertCollectable(tabWeakRef);
+        JMemoryBuddy.assertCollectable(textFieldWeakRef);
```

Comment:
perhaps the utility method operating on an array should accept varargs, like so:

`assertCollectable(WeakRef a, WeakRef ... b)`

(to differentiate it from a single arg method, and to make sure there are 2+ args)

Reply:
Done